contributing/team-communication.rst

```diff
@@ -6,35 +6,27 @@ how to get plugged in. There is a fairly extensive amount of
 communication flying around related to the project, and being able to
 find and track it may take some time.
 
-.. note:: Some of the links below go to private pages on openmicroscopy.org.
-    New hires in the core development team will be given
-    access to these automatically but you can also request a log-in as an
-    external collaborator.
-
 Instant messaging and video conferencing
 ----------------------------------------
 
-On a day-to-day level, the team meets in a Slack chatroom (see the list of
-:team_plone:`addresses <addresses>` for the specifics). Slack can be used in
-your internet browser or via an app; you will be invited to join the team by
-an admin.
+On a day-to-day level, the team meets in a Slack chatroom. Slack can be used
+in your internet browser or via an app; you will be invited to join the team
+by an admin.
 
 The daily stand-up meeting is managed via the '#general' channel, with notes
 in google docs that are edited throughout the day as people complete the tasks
 assigned to them.
 
-Slightly less frequently, the team meets on Zoom for voice discussions. These
-subgroup, or “mini group”, meetings are organized as needed, but should
-provide feedback (tickets, notes, etc). More information is available under
-:community_plone:`conference-calls <minutes/conference-calls>`.
+Slightly less frequently, members of the team meet on Zoom for voice
+discussions. These meetings are organized as needed, but should provide
+feedback where appropriate (tickets, notes, etc).
 
 Other IM tools
 ^^^^^^^^^^^^^^
 
 Slack is the only IM tool used by the entire OME team. Some team members do
-also use IRC (#ome on irc.freenode.net) and provide support via that channel
-but all significant conversations should be documented formally via Trac
-tickets or Trello. In general, all external requests for help are best
+also use IRC (#ome on irc.freenode.net) and **may** provide support via that
```

Comment:
italics alone should be sufficient emphasis here?

Reply:
I tend to prefer bold, italics doesn't draw the eye the same, I find it more suited to fiction where you expect the reader to be paying thorough attention throughout ;)